api/app/tests/apps/api/v1/test_history_endpoint.py

```diff
@@ -169,6 +176,15 @@ def test_sla_only_once_in_history(self):
         self.assertEqual(self.signal.category_assignments.all().order_by('created_at')[0].category.handling_message,
                          sla_description_in_history[0])
 
+    def test_history_no_permissions(self):
+        """
+        The sia_read_user does not have a link with any department and also is not configured with the permission
+        "sia_can_view_all_categories". Therefore it should not be able to see a Signal and it's history.
```

Comment:
We should probably add some checks that these properties of the `sia_read_user` remain true (because if the base class changes these tests might start passing "by accident").